src/status_im2/common/home/actions/view.cljs

```diff
@@ -50,27 +51,31 @@
 
 (defn clear-history-action
   [{:keys [chat-id] :as item}]
-  (hide-sheet-and-dispatch
+  (rf/dispatch
    [:bottom-sheet/show-sheet
     {:content (fn []
                 (confirmation-drawer/confirmation-drawer
                  {:title       (i18n/label :t/clear-history?)
                   :description (i18n/label :t/clear-history-confirmation-content)
                   :context     item
                   :button-text (i18n/label :t/clear-history)
-                  :on-press    #(hide-sheet-and-dispatch [:chat.ui/clear-history chat-id])}))}]))
+                  :on-press    #(do
+                                  (close-bottom-sheet-fn nil)
+                                  (rf/dispatch [:chat.ui/clear-history chat-id]))}))}]))
 
 (defn delete-chat-action
   [{:keys [chat-id] :as item}]
-  (hide-sheet-and-dispatch
+  (rf/dispatch
    [:bottom-sheet/show-sheet
     {:content (fn []
                 (confirmation-drawer/confirmation-drawer
                  {:title       (i18n/label :t/delete-chat?)
                   :description (i18n/label :t/delete-chat-confirmation)
                   :context     item
                   :button-text (i18n/label :t/delete-chat)
-                  :on-press    #(hide-sheet-and-dispatch [:chat.ui/remove-chat chat-id])}))}]))
+                  :on-press    #(do
```

Comment:
https://github.com/bbatsov/clojure-style-guide#no-multiple-forms-in-function-literals